### DISCUSSION TITLE
Full text search does not work, php error

### DISCUSSION THREAD

**Discussion by a-mian-lateral** [chamilo/chamilo-lms#5190] [General]
Hello, I have a question about the Xapian module, we installed the extension on the server and enabled the full text search but when we try to search something we get this error

`Fatal error: require_once(): Failed opening required 'xapian.php' (include_path='.:/usr/local/lib/php') in /var/www/chamilo/main/inc/lib/search/xapian/XapianQuery.php on line 7`

We gave a look at that file and noticed that it includes a file named "xapian.php" that does not exists, what are we missing?

**Comment by ywarnier** [comment]
You're not missing anything. We've got to update the fulltext search plugin. There's an issue about that but we keep pushing it further back because of a lack of time. Currently, it's planned to re-integrate it into Chamilo 2.1 which should be around the end of 2024. Sorry for the lack of information regarding this issue.

Of course, you are welcome to contribute, in code directly as a PR or financially through (ideally) one of the official providers of Chamilo (https://chamilo.org/en/providers/) but in general the number of people using it was very small before, not many people complained it wasn't working anymore (and it's like that since 4-5 years, I think) so it's not a big priority (although sincerely, I love the features it adds - it's just that there are so many other things to do that are also late here...).
